scania-digital-design-system/sdds
# feat(textarea/textfield): migrate textarea and textfield components

**Describe pull-request**  
Migration of the textfield and the textarea components.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#2409](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2409)

**How to test**  
1. Go to storybook link below
2. Check in components -> textarea/textfield
3. Check that the design is aligned with the one in Figma. 

**Screenshots**  
-

**Additional context**  
-
